scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl

```diff
@@ -10,18 +12,24 @@ workflow GvsCreateVATfromVDS {
         String filter_set_name
         String? vat_version
 
+        Int effective_scatter_count = 10
```

Comment:
do we want this to be a param rather than hard coded? am I missing something?

Reply:
It is a parameter, just defaults to 10.